### PR TITLE
Enable APM features by default

### PR DESCRIPTION
**Pull request by marandaneto**
## :bulb: Motivation and Context

Closes https://github.com/getsentry/sentry-dart/issues/1173

## :pencil: Checklist

- [X] I reviewed submitted code
- [X] I added tests to verify changes
- [ ] I updated the docs if needed
- [X] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: flutter/README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ runApp(
 This adds performance tracing for all `AssetBundle` usages, where the `AssetBundle` is accessed with `DefaultAssetBunlde.of(context)`.
 This includes all of Flutters internal access of `AssetBundle`s, like `Image.asset` for example.
 Tracing for `AssetBundle.loadStructuredData()` is currently disabled.
-It's hidden by the `enableStructureDataTracing` flag and considered experimental. Using it could lead to bugs. We recognize the irony.
+It's hidden by the `enableStructuredDataTracing` flag and considered experimental. Using it could lead to bugs. We recognize the irony.
```
Comment (ueman) [inline flutter/README.md:105]: Is it still experimental even though it's enabled by default?

Comment (marandaneto) [reply flutter/README.md:105]: I removed the experimental notes, and never got any complaints.

Comment (ueman) [reply flutter/README.md:105]: I guess the flag can then be removed altogether?

Comment (marandaneto) [reply flutter/README.md:105]: We can remove it on v8 maybe, it's always nice having an opt-out for whatever reason, at least for a bit, maybe enabled by default uncover bugs, so people can opt-out quickly.

**Review by brustolin — APPROVED** [review]
Looks like CHANGELOG is wrong, but other than that LGTM.

**Comment by marandaneto** [comment]
> Looks like CHANGELOG is wrong, but other than that LGTM.

Good catch, fixed.

**Pull request merged**